### ISSUE TITLE
Missing update indicator for core updates

### ISSUE THREAD

**Issue by ocean90** [obenland/shiny-updates#241]
**Steps to reproduce:**

* Set language to German
* Change version.php: `$wp_version = '4.5.1';` and `$wp_local_package = 'de_DE';`
* Change the version of two plugin so you get an update notification for them (Note: It doesn't happen with only one plugin update.)
* Visit update-core.php and click Update All

Actual: The buttons for the first two core updates don't get the loading icon
Excepted: One button gets a loading icon. (The other gets disabled?)

**Screencast:**

![missing](https://cloud.githubusercontent.com/assets/617637/15804973/47d8ec38-2b1b-11e6-80ba-38f07771814c.gif)

As soon as the plugin updates are done the icon gets added:
<img width="1098" alt="" src="https://cloud.githubusercontent.com/assets/617637/15804983/934dcc88-2b1b-11e6-99bd-7e14eb95e822.png">

**Comment by swissspidy** [comment]
When clicking "Update All" we loop through all items and trigger `wp.updates.updateItem()` for them. When there are two core updates (en_US + localized), we only update the localized one.

The lines in question currently look like this:

```
if ( 1 < $( '.update-link[data-type="core"]' ).length && 'core' === $itemRow.data( 'type' ) && 'en_US' === $itemRow.data( 'locale' ) ) {
	return;
}
```

I agree that the second item's button should probably be disabled in that case.

No idea yet why this only happens in this specific scenario.

**Comment by swissspidy** [comment]
As mentioned in #247, it's not exactly a _bug_. The spinner is added when all other updates are done, because the core update is always at the end.

**Comment by obenland** [comment]
Fixed in #247.
